external/libtww/d/com/d_com_inf_game.h

```diff
@@ -309,14 +309,6 @@ inline void dComIfGs_setMagic(uint8_t amount) {
     g_dComIfG_gameInfo.info.getPlayer().getPlayerStatusA().setMagic(amount);
 }
 
-inline uint16_t dComIfGs_getRupee() {
-    return g_dComIfG_gameInfo.info.getPlayer().getPlayerStatusA().getRupee();
-}
-
-inline void dComIfGs_setRupee(uint16_t amount) {
-    g_dComIfG_gameInfo.info.getPlayer().getPlayerStatusA().setRupee(amount);
-}
-
```

Comment:
whats the reasoning to remove these?

Reply:
Fig mentioned a month ago functions that aren't real functions in the game should not be in libtww, so I figured I'd remove these functions since I was updating all the code that uses these functions anyway.

However, I now realize I might have misinterpreted what Fig was saying, so I can add them back if people want that.

Reply:
If we need to implement these again at any time, we can do so later, but for now, this looks okay